### ISSUE TITLE
Liquibase unexpected-changesets verbose flag requires value.

### ISSUE THREAD

**Issue by XDelphiGrl** [liquibase/liquibase#2016]
## Environment
**Liquibase Version**:
Liquibase 4.4.2

**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>
CLI

**Liquibase Extension(s) & Version**: 
N/A

**Database Vendor & Version**:
MySQL 8 (db platform irrelevant to bug)

**Operating System Type & Version**:
Windows 10

## Description
The verbose parameter for the unexpected-changesets command requires a value in 4.4.x; a value was not required in 4.5.3.

## Steps To Reproduce
* Use the changelogs included in the changelogs.zip: [changelogs.zip](https://github.com/liquibase/liquibase/files/6926185/changelogs.zip)
* Deploy the change set in unexpchangsets_1.xml
```liquibase --log-level INFO update --changelog-file unexpchangsets_1.xml```
* Execute liquibase unexpected-changesets with --verbose
 ```liquibase --log-level INFO unexpected-changesets --changelog-file unexpchangsets_2.xml --verbose```

## Actual Behavior
Passing `--verbose` without a parameter throws the error "Error parsing command line: Missing required parameter for option '--verbose' (PARAM)". 

## Expected/Desired Behavior
Liquibase unexpected-changesets can be executed by providing `--verbose` without a value.
The output is: 
```
 unexpected changes were found in testuser@172.17.0.1@jdbc:mysql://localhost:3308/testdb1?useSSL=false&allowPublicKeyRetrieval=true
unexpchangsets_1.xml::1::deployMe::Liquibase Pro User
```

## Screenshots (if appropriate)
N/A

## Additional Context
Workaround: Pass `--verbose=true` to the unexpected-changesets command.

**Comment by molivasdat** [comment]
Hi @XDelphiGrl Thanks for creating this issue.  We will add it to the list of issues to process.

**Comment by FBurguer** [comment]
looks like the issue has been fixed for liquibase `4.15`.
